osquery/tables/system/linux/startup_items.cpp

```diff
@@ -0,0 +1,110 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+
+#include <osquery/core.h>
+#include <osquery/filesystem/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+#include <osquery/utils/conversions/split.h>
+
+namespace fs = boost::filesystem;
+
+namespace osquery {
+namespace tables {
+
+const std::vector<std::string> systemItemPaths = {"/etc/xdg/autostart/"};
```

Comment:
nitpick, please use `kSystemItemPaths` (the naming-style convention), this is helpful for recognizing values that are global.